src/registrar/admin.py

```diff
@@ -50,13 +53,51 @@ class MyHostAdmin(AuditedAdmin):
     inlines = [HostIPInline]
 
 
+class DomainApplicationAdmin(AuditedAdmin):
+
+    """Customize the applications listing view."""
+
+    # Trigger action when a fieldset is changed
+    def save_model(self, request, obj, form, change):
+        if change:  # Check if the application is being edited
+            # Get the original application from the database
+            original_obj = models.DomainApplication.objects.get(pk=obj.pk)
+
+            if obj.status != original_obj.status and obj.status == "investigating":
+                if (
+                    original_obj.submitter is None
+                    or original_obj.submitter.email is None
+                ):
+                    logger.warning(
+                        "Cannot send status change (in review) email,"
+                        "no submitter email address."
+                    )
+                    return
+                try:
+                    print(
```

Comment:
**question**: We don't usually leave debugging print statements in. Would this be better as a `logging.info` message that email is being sent to this email address?